core/src/common.cpp

```diff
@@ -253,6 +253,24 @@ namespace irods::http
 		log::trace("{}: Client is authenticated.", __func__);
 		return {.client_info = std::move(*client_info)};
 	} // resolve_client_identity
+
+	auto get_port_from_url(boost::urls::url_view _url) -> std::optional<std::string> {
+		if (_url.has_port()) {
+			return _url.port();
+		}
+
+		switch (auto scheme_type{_url.scheme_id()}; scheme_type) {
+		case boost::urls::scheme::https:
+			return "443";
```

Comment:
Let's log a message stating which port will be used. It can be a debug or trace level message.